include/constants/std_script.h

```diff
@@ -572,14 +572,14 @@
 #define std_hiddenitem_w20_deepseascale                 8229
 #define std_hiddenitem_r45_r46_pp_max                   8230
 
-#define _std_safari                     8800
-#define std_safari_place_object         8800
-#define std_safari_remove_object        8801
-#define std_safari_steps_out            8802
-#define std_safari_balls_out            8803
-#define std_safari_storage_out          8804
-#define std_safari_enter                8805
-#define std_safari_exit                 8806
+#define _STD_SAFARI                     8800
+#define STD_SAFARI_PLACE_OBJECT         8800
+#define STD_SAFARI_REMOVE_OBJECT        8801
+#define STD_SAFARI_STEPS_OUT            8802
+#define STD_SAFARI_BALLS_OUT            8803
+#define STD_SAFARI_STORAGE_OUT          8804
+#define STD_SAFARI_ENTER                8805
+#define STD_SAFARI_EXIT                 8806
```

Comment:
Partial application of style change in file

Reply:
for these constants, would we want to capitlise them in the scripts too? or just in the C code

Reply:
Neither imo

Reply:
I can put it back, however do note that it would be inconsistent with every constant and the style for them